dep -v causes a panic

I'm going to fix this I'm just reminding myself in case I get distracted in the next ten minutes.

```console
$ ./dep -v
panic: runtime error: slice bounds out of range

goroutine 1 [running]:
main.main()
	/usr/local/google/home/jessfraz/.go/src/github.com/golang/dep/main.go:63 +0x347
```

--- a/main.go
+++ b/main.go
@@ -60,7 +60,9 @@ func main() {
 			}
 			args = cmd.flag.Args()
 		} else {
-			args = args[1:]
+			if len(args) > 0 {
+				args = args[1:]
+			}
 		}
 
 		if err := cmd.fn(args); err != nil {

